src/ripple/rpc/GRPCHandlers.h

```diff
@@ -71,6 +71,9 @@ std::pair<org::xrpl::rpc::v1::GetLedgerDiffResponse, grpc::Status>
 doLedgerDiffGrpc(
     RPC::GRPCContext<org::xrpl::rpc::v1::GetLedgerDiffRequest>& context);
 
+std::pair<org::xrpl::rpc::v1::GetAmmInfoResponse, grpc::Status>
+doAmmInfoGrpc(RPC::GRPCContext<org::xrpl::rpc::v1::GetAmmInfoRequest>& context);
+
```

Comment:
This is not needed. gRPC methods like this have been deprecated, no need to add more